src/Groups.ts

```diff
@@ -10,12 +10,160 @@ export interface Permissions {
   securityCategoryIds?: number[];
 }
 
+export type READ = 'READ';
+export type WRITE = 'WRITE';
+export type LIST = 'LIST';
+export type CREATE = 'CREATE';
+export type EXECUTE = 'EXECUTE';
+export type UPDATE = 'UPDATE';
+export type DELETE = 'DELETE';
+export type MEMBEROF = 'MEMBEROF';
+
+export interface GroupsAcl {
+  groupsAcl: {
+    actions?: (LIST | READ | CREATE | UPDATE | DELETE)[];
+    scope: {
+      all?: {};
+      currentuserscope?: {};
+    };
+  };
+}
+
+export interface AssetsAcl {
+  assetsAcl: {
+    actions?: (READ | WRITE)[];
+    scope: {
+      all?: {};
+    };
+  };
+}
+
+export interface EventsAcl {
+  eventsAcl: {
+    actions?: (READ | WRITE)[];
+    scope: {
+      all?: {};
+    };
+  };
+}
+
+export interface FilesAcl {
+  filesAcl: {
+    actions?: (READ | WRITE)[];
+    scope: {
+      all?: {};
+    };
+  };
+}
+
+export interface UsersAcl {
+  usersAcl: {
+    actions?: (LIST | CREATE | DELETE)[];
+    scope: {
+      all?: {};
+      currentuserscope?: {};
```

Comment:
What is the reason the name is currentuserscope and not currentUserScope?

Reply:
This is the name used in the REST api unfortunately.

@ngerstle-cognite Any reason for why this is not camel case?

Reply:
not that I recall off the top of my head. 
fixing it is an annoying breaking change, but could be done. It's just not worth the effort any time soon.